src/defaultPropTypes.js

```diff
@@ -17,11 +17,11 @@ export default {
   activeBarStyle: PropTypes.object,
   dismissAfter: PropTypes.oneOfType([
     PropTypes.bool,
-    PropTypes.number
+    PropTypes.number,
```

Comment:
@pburtchaell Do you want dangling commas?

@joeyfigaro This seems to be an unnecessary addition.